src/__tests__/booleanParser.test.js

```diff
@@ -0,0 +1,26 @@
+import { isBoolean, parse } from '../booleanParser';
```

Comment:
Most of the code changes are one of:
 - Proper function declarations and/or lambdas that pass eslint
 - Single quotes
 - Four space indent